lib/ansible/modules/cloud/vmware/vmware_vswitch.py

```diff
@@ -112,9 +114,9 @@ def __init__(self, module):
         self.content = None
         self.vss = None
         self.module = module
-        self.switch_name = module.params['switch_name']
+        self.switch_name = module.params['switch']
         self.number_of_ports = module.params['number_of_ports']
-        self.nic_name = module.params['nic_name']
+        self.nic_name = module.params['nics']
```

Comment:
So like I indicated on IRC, I would prefer to see self.nics being used instead of self.nic_name as it is more clear from the code that it is a list. Similarly I would also use self.switch over self.switch_name just because there is a 1-to-1 mapping between parameter name and internal variable.